### PR TITLE
`config.yaml`: Fix data lab configuration

### PR DESCRIPTION
**Pull request by razvand**
Fix data lab configuration

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -84,25 +84,25 @@ docusaurus:
...
-              location: content/chapters/compute/data
+              location: content/chapters/data/lab
               name: content/memory-support
           - Memory Security:
-              location: content/chapters/compute/data
+              location: content/chapters/data/lab
               name: content/memory-security
           - Arena:
-              location: content/chapters/compute/data
+              location: content/chapters/data/data
```
Comment (teodutu) [inline config.yaml:105]:
```suggestion
              location: content/chapters/data/lab
```

**Review by teodutu — CHANGES_REQUESTED** [review]
The path of the Arena section is wrong. Otherwise, the PR should be OK.

**Comment by razvand** [comment]
> The path of the Arena section is wrong. Otherwise, the PR should be OK.

@teodutu , fixed

**Review by teodutu — APPROVED** [review]
Should be alright now.

**Pull request merged**